src/styles/clock.scss

```diff
@@ -0,0 +1,47 @@
+.clock {
+  position: relative;
+  width: $clock-size;
+  height: $clock-size;
+  border: 4px dotted #48d1cc;
+  border-radius: 50%;
+
+  &__min-hand {
+    position: absolute;
+    z-index: 1;
+    background-color: $color-min;
+    border: 2px solid #000;
+    width: $min-hand-width;
+    height: $clock-size / 4;
+    left: calc(50% - #{$min-hand-width} / 2);
```

Comment:
to avoid this calculations and make element's behavior more predictable you can use pattern which center your element:

> top: 50%;
>   left: 50%;
>   transform: translate(-50%, -50%);